tasks/settings.yml

```diff
@@ -29,6 +29,14 @@
     line: '{{ jenkins_http_port_param }}={{ jenkins_http_port }}'
   register: jenkins_http_config
 
+- name: Ensure jenkins_home {{ jenkins_home }} exists
+  file:
+    path: "{{ jenkins_home }}"
+    state: directory
+    owner: jenkins
+    group: jenkins
+    mode: u+rwx
```

Comment:
Can you use the octal instead? (`0700`, I believe), and confirm this is the proper permissions setting for the directory? I can't find any official documentation on how permissions on JENKINS_HOME should be set :/